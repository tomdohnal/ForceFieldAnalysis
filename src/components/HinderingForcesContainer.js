// @flow
import React from 'react';

import ForcesContainer from '../containers/ForcesContainer';

const HinderingForcesContainer = () => (
  <ForcesContainer submitButtonText="Create Hindering Force" driving={false} />
);

export default HinderingForcesContainer;
