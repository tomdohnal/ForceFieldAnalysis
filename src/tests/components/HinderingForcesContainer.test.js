import React from 'react';
import { shallow } from 'enzyme';

import HinderingForcesContainer from '../../components/HinderingForcesContainer';

describe('HinderingForcesContainer', () => {
  it('renders correctly', () => {
    const tree = shallow(<HinderingForcesContainer text="Create Hindering Force" />);

    expect(tree).toMatchSnapshot();
  });
});
