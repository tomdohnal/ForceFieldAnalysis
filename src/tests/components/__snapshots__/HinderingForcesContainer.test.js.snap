// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HinderingForcesContainer renders correctly 1`] = `
<Connect(ForcesContainer)
  driving={false}
  submitButtonText="Create Hindering Force"
/>
`;
